android/app/src/androidTest/java/com/zingo/RPCModuleTests.kt

```diff
@@ -0,0 +1,115 @@
+package org.ZingoLabs.Zingo
+
+import android.util.Base64
+import java.io.InputStream
+import com.fasterxml.jackson.module.kotlin.jacksonObjectMapper
+import com.fasterxml.jackson.module.kotlin.readValue
+
+//test dependencies
+import com.google.common.truth.Truth.assertThat
+import org.junit.Test
+
+class RPCModule {
+    private external fun execute(cmd: String, args: String): String
+    private external fun initfromseed(serveruri: String, seed: String, birthday: String, saplingOutputb64: String, saplingSpendb64: String, datadir: String): String
```

Comment:
you can change this line for this...
`private external fun initfromseed(serveruri: String, seed: String, birthday: String, datadir: String): String`.

Reply:
Thanks now your PR landed I can make this change in the morning!